lib/rubocop/ast/node.rb

```diff
@@ -321,6 +321,10 @@ def single_line?
         !multiline?
       end
 
+      def two_lines?
```

Comment:
Perhaps instead of this very specific one, we can add a `#line_count` method? Then redefine `#single_line?` and `#multiline?` in terms of that, and in your cop:

```ruby
return if node.line_count > 2
```